zebra-chain/src/parameters/network/tests/vectors.rs

```diff
@@ -164,8 +185,11 @@ fn check_network_name() {
         "Mainnet",
         "Mainnet should be displayed as 'Mainnet'"
     );
-
-    // TODO: Check Regtest
+    assert_eq!(
+        Network::new_regtest(Default::default()).to_string(),
```

Comment:
This checks that `with_sapling_hrps()` will consider Mainnet/Testnet/Regtest HRPs as valid values as well.